osu.Game/Rulesets/UI/FrameStabilityContainer.cs

```diff
@@ -169,6 +164,13 @@ private void updateClock()
             }
             finally
             {
+                if (newProposedTime != manualClock.CurrentTime)
+                    direction = newProposedTime > manualClock.CurrentTime ? 1 : -1;
+
+                manualClock.CurrentTime = newProposedTime;
```

Comment:
This doesn't look correct, due to line 160.

Reply:
ah, that was on my mental check list but i forgot to fix it. good catch.